performance/font-loading.md

```diff
@@ -0,0 +1,59 @@
+# Font Loading
+
+## When to use webfonts
+Before using webfonts it's worth understanding what benefits you believe a webfont will provide your product and assess those against the potential impact they may have on things like accessibility or page speed. For example:
```

Comment:
```suggestion
Before using webfonts it's worth defining what benefits you believe a webfont will provide your product, and assessing those against the potential impact they may have on things like accessibility or page speed. For example:
```

Reply:
Have reworked this section now so no need to address this suggestion. Have committed your other suggestions though. Thanks!